.gitignore

```diff
@@ -0,0 +1 @@
+*.sublime-*
```

Comment:
Point taken but I can't add it to global ignore since I use git for other things than just evaporate. Is it really going to cause a problem for you? It's been fine in other repos.